dsc/workflows/base/simple_csv.py

```diff
@@ -20,6 +22,84 @@ class SimpleCSV(Workflow):
 
     workflow_name: str = "simple_csv"
 
+    def reconcile_bitstreams_and_metadata(
+        self, metadata_file: str = "metadata.csv"
+    ) -> None:
+        """Summary.
```

Comment:
Still need a docstring here

Reply:
@ehanson8 @ghukill What do you think of this docstring?

```python
"""Reconcile item metadata from metadata CSV file with bitstreams.

For SimpleCSV workflows, bitstreams (files) and a metadata CSV file
are uploaded to a designated batch folder on S3. The reconcile method
ensures that every bitstream on S3 has metadata--a row in the metadata
CSV file--associated with it and vice versa.
```